packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -290,6 +297,46 @@ export class ClusterReachability extends EventsScope {
     }
   }
 
+  /**
+   * Determines NAT Type.
+   *
+   * @param {RTCIceCandidate} candidate
+   * @returns {void}
+   */
+  private determineNatType(candidate: RTCIceCandidate) {
+    this.srflxIceCandidates.push(candidate);
+
+    if (this.srflxIceCandidates.length > 1) {
+      const portsFound: {[key: string]: Set<number>} = {};
+
+      this.srflxIceCandidates.forEach((c) => {
+        const key = `${c.address}:${c.relatedPort}`;
+        if (!portsFound[key]) {
+          portsFound[key] = new Set();
+        }
+        portsFound[key].add(c.port);
+      });
+
+      Object.entries(portsFound).forEach(([, ports]) => {
+        if (ports.size > 1) {
+          // Found candidates with the same address and relatedPort, but different ports
+          this.result.natType = NatType.SymmetricNat;
```

Comment:
we could also add support for other types of nat like full cone, right? (if we get multiple srflx candidates from different servers all with same port) It could be added in another PR...